src/agent/commands/actions.js

```diff
@@ -295,6 +295,14 @@ export const actionsList = [
             await skills.goToBed(agent.bot);
         })
     },
+    {
+        name: "!goToDeath",
```

Comment:
this command is unnecessary, as the bot can use goToPosition('last_death_position'). please remove this command/skill

Reply:
the bot can not just use "goToPosition('last_death_position')" as the "goToPosition()" skill requires its peramiters to be x, y, z and distance values so you cant just parse though the memory bank position.

Reply:
sorry! I meant goToPlace

Reply:
ok, done!